tests/conftest.py

```diff
@@ -43,7 +43,7 @@ def pytest_addoption(parser):
         help="Use PostgreSQL backend instead of default memory mock",
     )
     parser.addoption("--runslow", action="store_true", help="Don't skip slow tests")
-    parser.addoption("--runfuse", action="store_true", help="Don't skip FUSE/WinFSP tests")
+    parser.addoption("--runmountpoint", action="store_true", help="Don't skip FUSE/WinFSP tests")
```

Comment:
Maybe switch to `--skipmountpoint` so the mountpoint tests run by default, now that mountpoints are supported on both windows and linux?

Reply:
mountpoint tests are slow (and beside not thaaat stable right now), so I'd better keep it this way ;-)